app/views/deregister_exemptions/new.html.erb

```diff
@@ -48,7 +49,7 @@
             <span class="form-hint"><%= t(".labels.message_hint") %></span>
           </legend>
 
-          <fieldset id="message">
```

Comment:
W3C validation error was about multiple tags with the same `message` id. I have left the other one which is more relevant to have `message` as an id and removed this one.